addons/event/static/src/client_action/event_barcode.xml

```diff
@@ -4,34 +4,26 @@
 
     <t t-name="event.EventScanView">
         <div class="o_event_barcode_bg o_home_menu_background">
-            <div class="o_event_barcode_main bg-view">
-                <a href="#" class="o_event_previous_menu float-start"><i class="oi oi-chevron-left" t-on-click.prevent="() => this.onClickBackToEvents()"></i></a>
-                <div class="text-center">
-                    <h1 t-out="data.name"/>
-                    <p>
-                        <t t-if="data.city and data.country">
-                            <t t-out="data.city"/> - <t t-out="data.country"/>
-                        </t>
-                        <t t-if="data.city and !data.country" t-out="data.city"/>
-                        <t t-if="data.country and !data.city" t-out="data.country"/>
-                    </p>
-                    <h2><small>Welcome to</small> <t t-out="data.company_name"/></h2>
-                    <img t-if="data.company_id" t-attf-src="/web/image/res.company/{{data.company_id}}/logo_web" alt="Company Logo" class="o_event_barcode_company_image"/>
-                </div>
-                <div class="row">
-                    <div class="col-sm-5 mt16">
-                        <BarcodeScanner onBarcodeScanned="(ev) => this.onBarcodeScanned(ev)"/>
-                        <h5 class="mt8 mb0 text-muted">Scan a badge</h5>
+            <div class="o_event_barcode_main container d-flex flex-column h-100 h-sm-auto bg-view shadow">
+                <header>
```

Comment:
is the header main footer structure really needed ?

Reply:
No, we can replace them with divs, it just provides better readability.

Reply:
Ok let us go with divs then :)